vmdb/app/models/vm_or_template.rb

```diff
@@ -1064,7 +1064,7 @@ def storage2active_proxies(all_proxy_list = nil)
     # MiqServer coresident proxy needs to contact the host and provide credentials.
     # Remove any MiqServer instances if we do not have credentials
     rsc = self.scan_via_ems? ? self.ext_management_system : self.host
-    proxies.delete_if {|p| MiqServer === p} if rsc && rsc.authentication_invalid?
+    proxies.delete_if {|p| MiqServer === p} if rsc && rsc.missing_credentials?
```

Comment:
Squirrel/shiny object:  Note, it's really a shame we can't bubble up that these "proxies" were eliminated due to missing creds, bad authentication, bad response (down)... the user will end up with "no eligible proxies" and have no idea why...